pkg/consensus/consensus.go

```diff
@@ -328,7 +331,19 @@ func (s *service) getBlock(h util.Uint256) block.Block {
 
 func (s *service) getVerifiedTx(count int) []block.Transaction {
 	pool := s.Config.Chain.GetMemPool()
-	txx := pool.GetVerifiedTransactions()
+	txx := make([]*transaction.Transaction, 0, len(s.lastProposal))
```

Comment:
It only makes sense doing this inside `if s.dbft.ViewNumber > 0`, I think.

Reply:
Otherwise you're allocating some memory here and then just throwing it away ten lines below.